### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#271] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152790873-195aa322-2046-4c5c-ac72-085ad49f3eaf.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/216766696-e1405942-552c-48e4-8f39-f04919664dd5.png)

**Comment by LuginiAndrea** [comment]
1. Vero. Prenendo $\mathcal{P}\left(\mathbf{N}\right)$ e rimuovendo gli elementi di cardinalità infinita, ci rimangono sottoinsiemi di cardinalità finita, ovvero ci restano $\emptyset, N_1, N_2, ...$ così via all'infinito (senza mai raggiungerlo :P).
Facile inoltre vedere come questi insiemi, tolto $\emptyset$, siano rispettivamente $\subset$ di $N, N^2, ...$ quindi, sapendo che $\forall n \in \mathbf{N}, \mathbf{N}^n$ è numerabile, ne segue che anche questi insiemi sono numerabili. 
Inoltre questa "famiglia" di insiemi è numerabile, facile definire una biezione da $n \in \mathbf{N}$ a $N_n$.
Allora, sapendo che l'unione di una quantità numerabile di insiemi numerabili è numerabile, ne segue che il nostro insieme iniziale è numerabile.
2. Vero: basti pensare al fatto che esistono infinite funzioni suriettive $f: \mathbf{Z}\rightarrow \mathbf{Z}$. Ne segue che, in quanto $\mathbf{Z} \subset \mathbf{R}$, ne esistono infinite anche da $\mathbf{R} \rightarrow \mathbf{Z}$. 
3. No. Basti pensare a $f: \mathbf{Q} \rightarrow \mathbf{R}$, $f\left(x\right) = 1$
